geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/netty/Coder.java

```diff
@@ -78,18 +78,16 @@
   public static final byte[] bEMPTY_STRING = stringToBytes("$0\r\n\r\n");
 
   @MakeImmutable
-  public static final byte[] err = stringToBytes("ERR ");
+  public static final byte[] ERR = stringToBytes("ERR ");
 
   @MakeImmutable
-  public static final byte[] oom = stringToBytes("OOM ");
+  public static final byte[] MOVED = stringToBytes("MOVED ");
```

Comment:
After rebasing, could this be moved to the `StringBytesGlossary` class and renamed to be consistent with other constants in that class?

Reply:
Done.